UserViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UserViewController.swift
+//  Random Users
+//
+//  Created by Craig Belinfante on 9/13/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class UserViewController: UIViewController {
+    
+    var user: User?
+    
+    @IBOutlet weak var userLargeImage: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var phoneLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateViews()
+    }
+    
+    private func updateViews() {
+        guard let user = user else {return}
+        
+        nameLabel.text = user.name
+        phoneLabel.text = user.phone
+        emailLabel.text = user.email
+        
+        let url = URL(string: "\(user.large)")!
+        if let data = try? Data(contentsOf: url) {
```

Comment:
⚠️ DO NOT EVER DO THIS IN AN ACTUAL APP!!! I know this is a bit extreme with the all caps, but this is really, really, bad. This is the exact opposite of what the point of this concurrency sprint is all about. Data(contentsOf:) runs on the main thread, where the UI runs. Therefore, it blocks the UI while fetching the image. This is not at all what we want.

Fetching this image in your APIController would have been acceptable, as would using an operation. Please change this in your app at some point (when you have time, don't take away from what you are learning now).